src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -1073,6 +1073,33 @@ public static string GetMachineFQDN(string hostname)
             return fqdn.ToString();
         }
 
+        public static bool IsManagedSNI()
```

Comment:
I'll let you confirm, but there looks like's a function that already exists in this class that performs this operation:
public static bool IsUsingManagedSNI()